packages/lime-proto-olsr6/src/olsr6.lua

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/lua
+
+local network = require("lime.network")
+local config = require("lime.config")
+local fs = require("nixio.fs")
+local libuci = require("uci")
+local wireless = require("lime.wireless")
+local utils = require("lime.utils")
+
+olsr = {}
+
+olsr.configured = false
+
+function olsr.configure(args)
+	if olsr.configured then return end
+	olsr.configured = true
+
+	local uci = libuci:cursor()
+	local _, ipv6 = network.primary_address()
+
+    fs.writefile("/etc/config/olsrd6", "")
+
+	uci:set("olsrd6", "lime", "olsrd")
+	uci:set("olsrd6", "lime", "LinkQualityAlgorithm", "etx_ff")
+	uci:set("olsrd6", "lime", "IpVersion", "6")
+
+    uci:set("olsrd6", "limejson", "LoadPlugin")
+    uci:set("olsrd6", "limejson", "library", "olsrd_jsoninfo.so.0.0")
+    uci:set("olsrd6", "limejson", "accept", "::1")
+
+	uci:set("olsrd6", "limehna", "Hna6")
+	uci:set("olsrd6", "limehna", "netaddr", ipv6:network():string())
+	uci:set("olsrd6", "limehna", "prefix", ipv6:prefix())
+
+	uci:save("olsrd6")
+
+
+end
+
+function olsr.setup_interface(ifname, args)
+	if ifname:match("^wlan%d+_ap") then return end
+	vlanId = args[2] or 24
```

Comment:
default vlan-id of protocols should be < 16 see discussion at https://github.com/libre-mesh/lime-packages/pull/10